tensorflow/buildspec.yml

```diff
@@ -80,7 +80,7 @@
       device_type: &DEVICE_TYPE gpu
       python_version: &DOCKER_PYTHON_VERSION py3
       tag_python_version: &TAG_PYTHON_VERSION py37
-      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", "cu102", "-", *OS_VERSION ]
```

Comment:
hardcoding inference to cu102, since there is no cu101 inference container. #418 provides good tests for this